client/me/purchases/manage-purchase/style.scss

```diff
@@ -55,6 +55,21 @@
 		.manage-purchase__title {
 			width: 60%;
 		}
+
+		@include breakpoint( "<660px" ) {
+			.manage-purchase__detail {
+				width: 25%;
+			}
+
+			.manage-purchase__detail-label {
+				width: 70%;
+			}
+
+			.manage-purchase__detail,
+			.manage-purchase__detail-label {
+				height: 25px;
```

Comment:
These styles are specific to the mobile view. Do you mean that these should be the default styles, and that they should be overruled with different `>660px` styles?